cabal.haskell-ci

```diff
@@ -2,3 +2,13 @@ no-tests-no-benchmarks: False
 unconstrained:          False
 allow-failures:         <7.3
 irc-channels:           irc.freenode.org#haskell-lens
```

Comment:
Don't forget `irc-if-in-origin-repo: True`.

Reply:
done